packages/arb-token-bridge-ui/src/state/app/state.ts

```diff
@@ -44,6 +44,8 @@ export enum DepositStatus {
 }
 
 export interface MergedTransaction {
+  sender?: string
+  destination?: string
```

Comment:
Is it possible to have `destination` not be optional?

Reply:
Same as for `sender?`, results from SDK force these types on us.